tfe/resource_tfe_variable.go

```diff
@@ -234,7 +234,7 @@ func resourceTFEVariableRead(d *schema.ResourceData, meta interface{}) error {
 	// Update config.
 	d.Set("key", variable.Key)
 	d.Set("category", string(variable.Category))
-	d.Set("description", string(variable.Description))
+	d.Set("description", variable.Description)
```

Comment:
removes redundant string conversion, since `Description` is of type `string` in the struct